azavea/cac-tripplanner
# Feature/kak/update dependencies

## Overview

Update project dependencies for ansible roles, python packages, and Javascript packages.


### Notes

Turf exports have changed to provide a top-level `default` function instead of simply exporting the function. I couldn't find a straightforward way to transform the `browserify` build to rename the export, so the calls to `turf` `distance` and `pointOnLine` have changed instead.

Attempting to update the Postgresql support ansible role version resulted in provisioning failures, perhaps due to the role changing to default to 9.5, so that role is behind a version still.


## Testing Instructions

 * Rebuild VMs (may need to destroy first, due to dependency changes)
 * Site should still function as expected
 * To test turf `distance` functionality: set origin on home page; distances should populate on destinations
 * To test turf 'pointOnLine`: drag a trip linestring to add intermediate stops

Connects azavea/urban-apps#173
